crates/bevy_sprite/src/render/mod.rs

```diff
@@ -166,6 +171,21 @@ pub struct ExtractedSprite {
 #[derive(Default)]
 pub struct ExtractedSprites {
     pub sprites: Vec<ExtractedSprite>,
+    pub indexes: Vec<ExtractedSpriteIndex>,
+}
+
+impl ExtractedSprites {
+    pub fn push(&mut self, sprite: ExtractedSprite) {
+        self.indexes.push(ExtractedSpriteIndex {
+            handle: sprite.handle.clone_weak(),
+            w: sprite.transform.w_axis[2],
```

Comment:
This is actually the Z coordinate (which is the correct one to use, as before), not the W coordinate. The struct field should be called `z`.

Reply:
removed as it was slower